swagger.yaml

```diff
@@ -264,6 +264,412 @@ paths:
+      parameters:
+        - in: header
+          name: Columbus-User-Email
+          type: string
+          format: email
+          required: true
+        - in: query
+          name: type
+          schema:
+            type: string
+            enum:
+              - openeded
+              - issues
+              - qanda
+              - all
+            default: all
+        - in: query
+          name: state
+          schema:
+            type: string
+            enum:
+              - open
+              - closed
+              - all
+            default: open
+        - in: query
+          name: owner
+          type: string
+        - in: query
+          name: sort
+          schema:
+            type: string
+            enum:
+              - created_at
+              - updated_at
+            default: created_at
+        - in: query
+          name: direction
+          schema:
+            type: string
+            enum:
+              - asc
+              - desc
+            default: desc
+        - in: query
+          name: assignee
+          type: string
+        - in: query
+          name: asset
+          type: string
+        - in: query
+          name: labels
+          type: string
+          description: comma separated value of label
+        - in: query
+          name: size
+          type: integer
+        - in: query
+          name: offset
+          type: integer
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/Discussion"
+        400:
+          description: validation error
+          schema:
+            $ref: "#/definitions/ValidationError"
+        500:
+          description: "internal server error"
+          schema:
+            $ref: "#/definitions/InternalServerError"
+    post:
+      tags:
+        - Discussions
+      summary: Create a new discussion
+      description: "Create a new discussion"
+      produces:
+        - "application/json"
+      parameters:
+        - in: header
+          name: Columbus-User-Email
+          type: string
+          format: email
+          required: true
+        - in: body
+          name: ""
+          required: true
+          schema:
+            allOf:
+              - $ref: "#/definitions/Discussion"
+              - required:
+                  - title
+                  - body
+                  - type
+      responses:
+        201:
+          description: Created
+          schema:
+            type: object
+            properties:
+              id:
+                type: integer
+                description: id of the newly created discussion
+        400:
+          description: validation error
+          schema:
+            $ref: "#/definitions/ValidationError"
+        500:
+          description: "internal server error"
+          schema:
+            $ref: "#/definitions/InternalServerError"
+  "/v1beta1/discussions/{id}":
```

Comment:
this is duplicate key, cannot run swagger due to this.

Reply:
Thanks stew, fixing this @StewartJingga